cmd/sops/main.go

```diff
@@ -1051,11 +1055,32 @@ func keyservices(c *cli.Context) (svcs []keyservice.KeyServiceClient) {
 }
 
 func inputStore(context *cli.Context, path string) common.Store {
-	return common.DefaultStoreForPathOrFormat(path, context.String("input-type"))
+	var configPath string
+	if context.String("config") != "" {
+		configPath = context.String("config")
+	} else {
+		// Ignore config not found errors returned from FindConfigFile since the config file is not mandatory
+		configPath, _ = config.FindConfigFile(".")
+	}
+	storesConf, _ := config.LoadStoresConfig(configPath)
```

Comment:
It looks to me that lines 1058 to 1065 could be moved to a function, since they appear twice in this file (here and in `outputStore()`).

Reply:
Changed